tests_integration/__tests__/early-exit.js

```diff
@@ -85,3 +85,19 @@ describe("throw error and show usage with something unexpected", () => {
     status: "non-zero",
   });
 });
+
+describe("node version error", () => {
+  const originalProcessVersion = process.version;
+  try {
+    Object.defineProperty(process, "version", {
+      value: "v8.0.0",
+      writable: false,
+    });
+    runPrettier("cli", ["--help"]).test({ status: 1 });
+  } finally {
+    Object.defineProperty(process, "version", {
+      value: originalProcessVersion,
+      writable: false,
+    });
+  }
+});
```

Comment:
I think we need test exit code here too

Reply:
That's what `test({ status: 1 });` does.

Reply:
I should be a little more attentively sometimes, sorry